tests/src/Interop/CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
 
+if(WIN32)
+	if(CLR_CMAKE_HOST_ARCH STREQUAL arm)
+	    list(APPEND LINK_LIBRARIES_ADDITIONAL
```

Comment:
NIt

I don't see LINK_LIBRARIES_ADDITIONAL being defined anywhere is it a pre defined property in cmake ?